pkg/tbtc/coordination_test.go

```diff
@@ -0,0 +1,118 @@
+package tbtc
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/keep-network/keep-core/internal/testutils"
+)
+
+func TestCoordinationWindow_ActivePhaseEndBlock(t *testing.T) {
+	window := newCoordinationWindow(900)
+
+	testutils.AssertIntsEqual(
+		t,
+		"active phase end block",
+		980,
+		int(window.activePhaseEndBlock()),
+	)
+}
+
+func TestCoordinationWindow_EndBlock(t *testing.T) {
+	window := newCoordinationWindow(900)
+
+	testutils.AssertIntsEqual(
+		t,
+		"end block",
+		1000,
+		int(window.endBlock()),
+	)
+}
+
+func TestCoordinationWindow_IsAfterActivePhase(t *testing.T) {
```

Comment:
Shouldn't it be:

```suggestion
func TestCoordinationWindow_IsAfter(t *testing.T) {
```

?

Reply:
Yeah! Copilot tricked me 🤦